resources/user_config.ts

```diff
@@ -78,6 +78,7 @@ export type ConfigEntry = {
     options: BaseOptions,
     isDefault: boolean,
   ) => ConfigValue | void | undefined;
+  comment?: LocaleText;
```

Comment:
I agree with you about the empty string being slightly odd. I preferred this the way it was before when it was `Partial<LocaleText>`. I think most comments are not going to have an English entry, and so the empty string seems unnecessary a little.

Reply:
Updated! 'comment' type has been reverted to Partial<LocaleText>.